`svelte-kit package` crashes with `Cannot read property 'replace' of undefined` when preprocessor set in version `1.0.0-next.173`

### Describe the bug

Running `svelte-kit package` with typescript enabled and `1.0.0-next.173` causes a crash.

Works on version `1.0.0-next.172`.

### Reproduction

* `npm init svelte@next something` TypeScript **Yes**, everything else **No**
* `npm install`
* `npm install -D svelte2tsx typescript`
* create `src/lib/Test.svelte` with `<script></script>`
* add `"package": "svelte-kit package"` as package.json script
* `npm run package`

Repo if needed: https://github.com/SarcevicAntonio/as-ds

### Logs

```shell
> Cannot read property 'replace' of undefined
TypeError: Cannot read property 'replace' of undefined
    at file:///Users/sarcevic/dev/as-ds/node_modules/@sveltejs/kit/dist/chunks/index6.js:15455:16
    at String.replace (<anonymous>)
    at strip_lang_tag (file:///Users/sarcevic/dev/as-ds/node_modules/@sveltejs/kit/dist/chunks/index6.js:15451:21)
    at strip_lang_tags (file:///Users/sarcevic/dev/as-ds/node_modules/@sveltejs/kit/dist/chunks/index6.js:15439:2)
    at make_package (file:///Users/sarcevic/dev/as-ds/node_modules/@sveltejs/kit/dist/chunks/index6.js:15312:7)
    at async file:///Users/sarcevic/dev/as-ds/node_modules/@sveltejs/kit/dist/cli.js:923:4
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! as-ds@0.0.1 package: `svelte-kit package`
npm ERR! Exit status 1
```


### System Info

```shell
npx: Installierte 1 in 1.46s

  System:
    OS: macOS 11.5.2
    CPU: (8) x64 Intel(R) Core(TM) i5-8279U CPU @ 2.40GHz
    Memory: 291.90 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.17.0 - ~/.nvm/versions/node/v14.17.0/bin/node
    Yarn: 1.21.1 - ~/.yarn/bin/yarn
    npm: 6.14.13 - ~/.nvm/versions/node/v14.17.0/bin/npm
  Browsers:
    Chrome: 94.0.4606.61
    Firefox: 92.0
    Safari: 14.1.2
  npmPackages:
    @sveltejs/kit: next => 1.0.0-next.173 
    svelte: ^3.42.6 => 3.43.0 
```


### Severity

annoyance

### Additional Information

I have no clue and never worked with svelte/kit internals, but after some digging it seems to be related to https://github.com/sveltejs/kit/pull/2486/files.

If a preprocessor is set the `strip_lang_tag` function doesn't check if `attributes` is defined and calls `.replace(...)` on it, i.e. it expects every `script` and `style` tag to contain `lang` or `type` attributes.

Quickfix could be to add a `if (!attributes) return tag;` above `const idx = tag.indexOf(attributes);` in line 209 of `packages/kit/src/packaging/index.js`.

Workaround would be to add lang="" to any `script` and `style` tags which have not set anything.

--- a/.changeset/spotty-cooks-happen.md
+++ b/.changeset/spotty-cooks-happen.md
@@ -1,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+Fix script and style tags without attributes crashing svelte-kit package

--- a/packages/kit/src/packaging/index.js
+++ b/packages/kit/src/packaging/index.js
@@ -207,6 +207,7 @@ function strip_lang_tags(content) {
 			'g'
 		);
 		content = content.replace(regexp, (tag, attributes) => {
+			if (!attributes) return tag;
 			const idx = tag.indexOf(attributes);
 			return (
 				tag.substring(0, idx) +

--- a/packages/kit/src/packaging/test/fixtures/typescript/expected/Plain.svelte
+++ b/packages/kit/src/packaging/test/fixtures/typescript/expected/Plain.svelte
@@ -1,0 +1,4 @@
+<script>
+	/** @type {import('./foo').Foo} */
+	export let foo;
+</script>

--- a/packages/kit/src/packaging/test/fixtures/typescript/expected/Plain.svelte.d.ts
+++ b/packages/kit/src/packaging/test/fixtures/typescript/expected/Plain.svelte.d.ts
@@ -1,0 +1,26 @@
+/** @typedef {typeof __propDef.props}  PlainProps */
+/** @typedef {typeof __propDef.events}  PlainEvents */
+/** @typedef {typeof __propDef.slots}  PlainSlots */
+export default class Plain extends SvelteComponentTyped<
+	{
+		foo: boolean;
+	},
+	{
+		[evt: string]: CustomEvent<any>;
+	},
+	{}
+> {}
+export type PlainProps = typeof __propDef.props;
+export type PlainEvents = typeof __propDef.events;
+export type PlainSlots = typeof __propDef.slots;
+import { SvelteComponentTyped } from 'svelte';
+declare const __propDef: {
+	props: {
+		foo: import('./foo').Foo;
+	};
+	events: {
+		[evt: string]: CustomEvent<any>;
+	};
+	slots: {};
+};
+export {};

--- a/packages/kit/src/packaging/test/fixtures/typescript/expected/package.json
+++ b/packages/kit/src/packaging/test/fixtures/typescript/expected/package.json
@@ -5,6 +5,7 @@
 	"type": "module",
 	"exports": {
 		"./package.json": "./package.json",
+		"./Plain.svelte": "./Plain.svelte",
 		"./Test.svelte": "./Test.svelte",
 		"./Test2.svelte": "./Test2.svelte",
 		"./utils": "./utils.js",

--- a/packages/kit/src/packaging/test/fixtures/typescript/src/lib/Plain.svelte
+++ b/packages/kit/src/packaging/test/fixtures/typescript/src/lib/Plain.svelte
@@ -1,0 +1,4 @@
+<script>
+	/** @type {import('./foo').Foo} */
+	export let foo;
+</script>

